index.html

```diff
@@ -4,42 +4,210 @@
+                <div class="year2"><li class="text23">2015</li></div>
+            </ul>
+            <div class="primary-text22">
+            <p> A daily selection of privately personalised reads; no accounts or sign-up required. </p>
+            </div>
+            <div class="tags">
+                <div class="tags1"><button class="tag-texts1">html</button></div>
+                <div class="tags2">
+                   <button class="tag-texts2">css</button>
+                </div>            
+                <div class="tags3">
+                   <button class="tag-texts3">javascript</button>
+                </div>            
+    
+            </div>
+            <div class="action2">
+                <div class="normal-button3">
+                    <button class="action-text3">see project</button>
+                </div>
+            </div>
+        </div>
+            
+        </div>
     </div>
-  </section>
+    <div class="page4">
+        <div class="snapshoot3">
+        <img src="./Snapshoot Portfolio4.svg" alt="snap">
+        <div class="left-block2">
+            <div class="primary-text21"></div>
+        <h1 class="project-title2">Tonic</h1>
+        <ul class="frame2">
+            <div class="client2"><li class="text21">canopy</li></div>
+            <div class="role2"><li class="text22">Back End Dev</li></div>
+            <div class="year2"><li class="text23">2015</li></div>
+        </ul>
+        <div class="primary-text22">
+        <p> A daily selection of privately personalised reads; no accounts or sign-up required. </p>
+        </div>
+        <div class="tags">
+            <div class="tags1"><button class="tag-texts1">html</button></div>
+            <div class="tags2">
+               <button class="tag-texts2">css</button>
+            </div>            
+            <div class="tags3">
+               <button class="tag-texts3">javascript</button>
+            </div>            
 
+        </div>
+        <div class="action2">
+            <div class="normal-button4">
+                <button class="action-text4">see project</button>
+            </div>
+        </div>
+    </div>
+        
+    </div>
+</div>
+<div class="about-myself">
+    <div class="top">
+        <h1 class="text-intro">About myself</h1>
+        <p class="text-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of 
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <div class="supporting-text">
+                    <h4>LET'S CONNECT</h4>
+                </div>
+                <div class="frame">
+                    <i class='bx bxl-github'></i>
+                    <i class='bx bxl-linkedin' ></i>
+                    <i class='bx bxl-twitter' ></i>
+                    <i class='bx bxl-gmail' ></i>
+                </div>
```

Comment:
- [ ] Please kindly use `ul` and `li` instead of using the `div` tag for a semantic reason

Reply:
Kindly I suggest you to follow the advice of the previous reviewer on this very topic